queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/alternate_name.py

```diff
@@ -62,32 +67,43 @@ def update_partner_change(
             end_date=None,
             identifier=legal_entity.identifier,
             name=to_legal_name,
-            name_type=AlternateName.NameType.OPERATING,
-            start_date=start_date,
-            registration_date=change_filing_rec.effective_date,
+            name_type=AlternateName.NameType.DBA,
+            start_date=alternate_name.start_date,
+            business_start_date=business_start_date,
+            state=BusinessCommon.State.ACTIVE,
+            entity_type=BusinessCommon.EntityTypes.PARTNERSHIP,
+            naics_key=alternate_name.naics_key,
+            naics_code=alternate_name.naics_code,
+            naics_description=alternate_name.naics_description,
```

Comment:
`state` and naics related fields do not need to be populated for GPs in `alternate_name` record